### PR TITLE
Increase JWT token expiration delta

### LINKED ISSUE #30: Increase JWT token expiration time
## Description

JWT (JSON Web Tokens) token should have a higher time expiration value.
It is expiring too early.

### Update
- [x] JWT configurations should be changed
- [x] JWT token expiration changes to 1 week

## Definition of Done
- [x] All of the required items are completed.
- [x] PR should be sent to the `gsoc18-code` branch.
- [x] Approval by 1 mentor.

## Estimation
1 hour

### PR DESCRIPTION
**Pull request by isabelcosta**
### Description

Increased JWT access token expiration delta to 1 week.

Fixes #30 

### Type of Change:

- Code
- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?

I logged a user, with a value for the expiration delta of 10 seconds, to make sure if I was changing the right value.
Then I changed the value to 1 week and validated the value had changed.

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

File: app/api/resources/user.py [hunk 1]
```diff
@@ -144,7 +144,11 @@ class LoginUser(Resource):
     @users_ns.expect(login_request_body_model)  # , skip_none=True
     def post(cls):
         """
-        Login user.
+        Login user
+
+        The user can login with username + password.
```
Comment (m-murad) [inline app/api/resources/user.py]: (username or email) + password

**Review by m-murad — CHANGES_REQUESTED** [review]

**Review by m-murad — APPROVED** [review]

**Review by Dilu9218 — APPROVED** [review]

**Comment by m-murad** [comment]
@isabelcosta
 Please resolve conflict

**Comment by isabelcosta** [comment]
@m-murad will solve these now.

**Comment by isabelcosta** [comment]
@m-murad @Dilu9218 Just fixed the merge conflicts and added a mini test condition, to test the expiration delta configuration for every env (Dev, Test and Prod)

**Comment by m-murad** [comment]
@isabelcosta 
Can we send the expiry date of the token along with the response?
Can you open an issue for that, and send a separate PR?

**Comment by isabelcosta** [comment]
@m-murad sure I can create the issue.

**Pull request merged**